pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/DirectMethodTest.java

```diff
@@ -142,6 +144,7 @@ private Map<String, Collection<FieldSettings>> constructFieldSettingsMap(@NonNul
         return settingsMap;
     }
```

Comment:
Runs second time test run which means that it is depending on other tests changing the database. Needs to be fixed to run on freshly loaded data.